ui/cypress.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# On single node WORKLOAD_IP = eth0
+WORKLOAD_IP=$(ip a show eth0 | grep -Po "inet \K[\d.]+")
+UI_NODEPORT=$(sudo kubectl --kubeconfig=/etc/kubernetes/admin.conf get svc metalk8s-ui -n kube-system -o jsonpath='{.spec.ports[0].nodePort}')
+
+# Dependency needed to install Cypress
+yum install -y xorg-x11-server-Xvfb
+yum install -y gtk2
+yum install -y libXtst*
+# provides libXss
+yum install -y libXScrnSaver*
+# provides libgconf-2
+yum install -y GConf2*
+# provides libasound
+yum install -y alsa-lib*
+yum install -y epel-release
+
+yum install -y gcc-c++ make
+curl -sL https://rpm.nodesource.com/setup_10.x | bash -
+yum install -y nodejs
+
+# Remove the package.json to avoid installing useless dependency
+rm package.json package-lock.json
```

Comment:
Mmmh not really fond of this, it really is a hack. Did you try something like this?

```
npm install --no-save --quiet --no-package-lock <your_pkg>
```

Reference: https://stackoverflow.com/questions/52786695/npm-install-always-installs-everything-from-package-json

Reply:
Thanks for the tips